packages/web-app-files/src/search/sdk/index.ts

```diff
@@ -0,0 +1,52 @@
+import { SearchProvider, SearchList, SearchPreview } from 'search/src/types'
+import Preview from './preview'
+import List from './list'
+import { EventBus } from 'web-pkg/src/event'
+import { Store } from 'vuex'
+import VueRouter from 'vue-router'
+import get from 'lodash-es/get'
+
+export default class Provider extends EventBus implements SearchProvider {
+  public readonly id: string
+  public readonly label: string
+  public readonly previewSearch: SearchPreview
+  public readonly listSearch: SearchList
+  private readonly store: Store<any>
+  private readonly router: VueRouter
+
+  constructor(store: Store<any>, router: VueRouter) {
+    super()
+
+    this.id = 'files.sdk'
+    this.label = 'Search all files ↵'
```

Comment:
Same here, please use `$gettext` here, so that the string gets picked up for transifex.